src/app/shared/models/view-mode.ts

```diff
@@ -0,0 +1,8 @@
+import { ProductsView } from "src/app/shared/models/products-view";
```

Comment:
Please find a better place to define this interfere, no need to create a separate file for that.